runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -4450,7 +4525,11 @@ private static StreamingDataflowWorkerTestParams.Builder builder() {
 
     abstract int localRetryTimeoutMs();
 
-    abstract int maxWorkItemCommitBytes();
+    abstract long maxWorkItemCommitBytes();
```

Comment:
just use operational limits instead of separating the fields?

Reply:
Done.